pkg/resource/autonaming/config.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2016-2024, Pulumi Corporation.
```

Comment:
```suggestion
// Copyright 2024, Pulumi Corporation.
```
Same for any new files I think